pdal/util/FileUtils.cpp

```diff
@@ -305,9 +323,9 @@ std::string toCanonicalPath(std::string filename)
 
 #ifdef _WIN32
     filename = addTrailingSlash(filename);
-    char buf[MAX_PATH];
-    if (GetFullPathName(filename.c_str(), MAX_PATH, buf, NULL))
-        result = buf;
+    wchar_t buf[MAX_PATH];
+    if (GetFullPathNameW(toNative(filename).c_str(), MAX_PATH, buf, NULL))
```

Comment:
yes that breaks mingw build because toNative returns char* instead of wchar* there